packages/devtools/lib/src/framework/framework_core.dart

```diff
@@ -26,13 +28,35 @@ class FrameworkCore {
     theme.initializeTheme(uri.queryParameters['theme']);
 
     _setGlobals();
+    _setDebugFlags();
   }
 
   static void _setGlobals() {
     setGlobal(ServiceConnectionManager, ServiceConnectionManager());
     setGlobal(MessageBus, MessageBus());
   }
 
+  static void _setDebugFlags() {
+    if (window.location.search.isEmpty) {
+      return;
+    }
+    final queryParams = Uri.parse(window.location.toString()).queryParameters;
+
+    // TODO(kenzie): register a service on the devtools server that will dump
+    // all debug files behind the [debugTimeline] and [debugCpuProfile] flags.
+    // If we could automate this into some sort of bug report, we can probably
+    // remove this query param all together, but for now enabling manual dumps
+    // is fine.
+    final _debugTimeline = queryParams['debugTimeline'];
+    if (_debugTimeline != null && _debugTimeline == 'true') {
```

Comment:
these != null checks are not needed.

Reply:
simplify to
debugTimeline = queryParams['debugTimeline] == 'true';

Reply:
done.